src/pykiso/lib/auxiliaries/udsaux/common/odx_parser.py

```diff
@@ -24,9 +24,10 @@
 from enum import Enum
 from pathlib import Path
 from typing import List
-from xml.etree import ElementTree
 from xml.etree.ElementTree import Element
 
+from defusedxml import ElementTree
```

Comment:
Can you comment why changing the lib?

Reply:
XML is not secure against maliciously constructed data and it was recommended to use defusedxml instead.

Reply:
Please comment on it in the code